src/zx_graph.jl

```diff
@@ -17,10 +19,45 @@ struct ZXGraph{T<:Integer, P}
     st::Dict{T, SType}
 end
 
+has_edge(zxg::ZXGraph, vs...) = has_edge(zxg.mg, vs...)
+nv(zxg::ZXGraph) = nv(zxg.mg)
+ne(zxg::ZXGraph) = ne(zxg.mg)
+outneighbors(zxg::ZXGraph, v::Integer) = outneighbors(zxg.mg, v)
+inneighbors(zxg::ZXGraph, v::Integer) = inneighbors(zxg.mg, v)
+neighbors(zxg::ZXGraph, v::Integer) = neighbors(zxg.mg, v)
+rem_edge!(zxg::ZXGraph, v1::Integer, v2::Integer) = rem_edge!(zxg.mg, v1, v2, mul(zxg.mg, v1, v2))
+function add_edge!(zxg::ZXGraph, v1::Integer, v2::Integer, edge_type::Int = hadamard_edge)
```

Comment:
just use a `Symbol` here instead of integer, e.g `edge_type = :hadamard`, so we can have better error msg. no need to define a global constant just for this.